### PR TITLE
Copy links on click

### PR DESCRIPTION
**Pull request by AndCotOli**
Now, when a link is clicked, it's `href` value is copied to the clipboard.
I also added [Github Flavored Markdown](https://github.github.com/gfm/).

### CODE REVIEW

File: preload.js [hunk 1]
```diff
@@ -4,6 +4,30 @@ const timeago = require('timeago.js');
 const marked = require('marked');
 const createDOMPurify = require('dompurify');
 
+let markdownRenderer = marked.Renderer();
+markdownRenderer.link = (href, title, text) => {
+  let link = marked.Renderer.prototype.link.call(this, href, title, text);
+  link.addEventListener('click', () => {
```
Comment (vrumger) [inline preload.js]: How about adding the event listener on the `body` and checking the target?

**Comment by AndCotOli** [comment]
After thinking a little bit about it, I thought it'd be better to use [electron's clipboard API](https://www.electronjs.org/docs/api/clipboard).

Thanks @YouTwitFace for your feedback ❤️!

**Comment by vrumger** [comment]
I think you're misunderstanding what I'm talking about. Instead of `link.addEventListener('click', ...)` for every link, you can use:

```javascript
document.body.addEventListener('click', e => {
  const { target } = e;
  if (target.tagName === 'A') {
    e.preventDefault();
    clipboard.writeText(target.href);
  }
});
```

**Comment by w3cj** [comment]
I pushed some updates that change how images are rendered. Instead of trying to resolve conflicts, would you like to just close and re-open this?

**Comment by AndCotOli** [comment]
Sure.

**Comment by AndCotOli** [comment]
> 
> 
> I think you're misunderstanding what I'm talking about. Instead of `link.addEventListener('click', ...)` for every link, you can use:
> 
> ```js
> document.body.addEventListener('click', e => {
>   const { target } = e;
>   if (target.tagName === 'A') {
>     e.preventDefault();
>     clipboard.writeText(target.href);
>   }
> });
> ```

Oh, ok, got it now.
You're right, my approach didn't cover links created using `<a>` tags.

I'll implement it asap.

**Pull request closed without merging**